nrpy/infrastructures/BHaH/MoLtimestepping/MoL.py

```diff
@@ -474,7 +474,7 @@ def register_CFunction_MoL_step_forward_in_time(
         0
     ]  # Get the desired Butcher table from the dictionary
     num_steps = (
-        len(Butcher) - 1
+        len(Butcher) - 1 if Butcher[-1][0] == "" else len(Butcher) - 2
```

Comment:
Could you please explain what the conditional enables? If it's for supporting adaptive RK methods, then I think we should instead do a check and error out, as we don't currently support adaptive RK methods.

Reply:
For non-adaptive tables, the last row has a single list of coefficients and retains the `""` in the first cell whereas for schemes with adaptive ordering, the last two rows have a `LO` and a `HO` title.  This logic determines the number of steps based on if it's dealing with an adaptive scheme with needing to rely on the name of the Butcher table

Reply:
Okay, in that case before the conditional, MoL should instead have a check for whether the Butcher table is adaptive, and if so error out. This will enable us to restore the simpler logic. Again, adaptive timestepping is not supported in MoL, as adding it would require a significant adjustment to the overall algorithm.

Reply:
(we added adaptive timestepping for our separate ODE solver, not for BH@H... at least not yet)